lib/assert.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * Debug assert
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef LIB_ASSERT_H_
+#define LIB_ASSERT_H_
+
+
+void __lib_assert(int condition, const char *func, int line, const char *fmt, ...);
```

Comment:
why does this function's name begin with double underscore?

Reply:
Fixed